src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AddDefendantLitigationFriendCallbackHandler.java

```diff
@@ -61,10 +73,96 @@ private CallbackResponse aboutToSubmit(CallbackParams callbackParams) {
             .build();
     }
 
+    private CallbackResponse aboutToSubmit_multiparty(CallbackParams callbackParams) {
+        if (!featureToggleService.isMultipartyEnabled()) {
+            return aboutToSubmit(callbackParams);
```

Comment:
If we return the aboutToSubmit() then we are not populating the `respondent1LitigationFriend` field
As your CCD config is setup that we capture the details in `genericLitigationFriend` 

See comment: https://github.com/hmcts/civil-ccd-definition/pull/272/files#r767841372